src/components/SelectComponents/Autosuggest/Autosuggest.tsx

```diff
@@ -83,6 +89,9 @@ export function Autosuggest<S>({
     customSelectionIndicator = undefined,
     initInputValue,
     inputAttrs = {},
+    isFocused,
+    iconNode,
+    customListRender,
```

Comment:
Could you please add tests for the new props?

Reply:
:+1: